frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -404,10 +404,10 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                               to: isUsingSidebar ? undefined : urls.persons(),
                           },
                           {
-                              identifier: Scene.Events,
+                              identifier: Scene.Activity,
```

Comment:
@Twixes should this be part of the feature flag?

Reply:
What do you mean by "this" here in the navbar?

Reply:
This should go to the activity scene (with event explorer and live events tabs) if they have the feature flag and event explorer if they don't.

Reply:
Ah, right, that's exactly how `ActivityScene` has worked already before my tweaks - so we don't need to have Explore and Live as separate scenes.